RNTester/js/ListViewGridLayoutExample.js

```diff
@@ -30,28 +37,30 @@ var THUMB_URLS = [
   require('./Thumbnails/victory.png'),
 ];
 
-var ListViewGridLayoutExample = createReactClass({
-  displayName: 'ListViewGridLayoutExample',
+type State = {|
+  dataSource: ListViewDataSource,
+|};
 
-  statics: {
-    title: '<ListView> - Grid Layout',
-    description: 'Flexbox grid layout.',
-  },
+class ListViewGridLayoutExample extends React.Component<{}, State> {
```

Comment:
Can we use `RNTesterProps` here as well?

Reply:
Sure !